src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingPassBase.cs

```diff
@@ -630,16 +631,24 @@ bool IsInHtmlTag()
                     n => n is MarkupStartTagSyntax || n is MarkupTagHelperStartTagSyntax || n is MarkupEndTagSyntax || n is MarkupTagHelperEndTagSyntax);
             }
 
+            bool IsInDirectiveWithNoKind()
```

Comment:
Just to clarify, this captures `using`, `addTagHelper`, `removeTagHelper` and `tagHelperPrefix` right?

Reply:
Yes, and possibly others. This is a little bit of a speculative change to separate those from the others (where DirectiveKind == SingleLine) because I hope to remove that block eventually.